metpy/xarray.py

```diff
@@ -5,20 +5,22 @@
 from __future__ import absolute_import
 
 import functools
+import warnings
 
 import xarray as xr
 from xarray.core.accessors import DatetimeAccessor
 
-from .units import DimensionalityError, units
+from .units import DimensionalityError, UndefinedUnitError, units
 
 __all__ = []
 
 
-# TODO: Should we be providing renaming/mapping for easy access to x/y coordinates?
 @xr.register_dataarray_accessor('metpy')
 class MetPyAccessor(object):
     """Provide custom attributes and methods on XArray DataArray for MetPy functionality."""
 
+    nice_to_cf_axis_map = {'time': 'T', 'vertical': 'Z', 'y': 'Y', 'x': 'X'}
```

Comment:
A nit, but I'd probably call this something like `readable_to_cf_names` or similar. I've started trying to avoid `map` as part of variable names as it sometimes is redundant or even confusing in some contexts.